homework/homework_part_2/js/app.js

```diff
@@ -0,0 +1,75 @@
+// 1. The difference between interpolation and concatenation is the syntax.
+// 	  Interpolation looks like this: console.log(`This is how ${interpolation} works')
+// 	  Concatenation looks like this: console.log('We ' + 'can ' + concatenate ' like this!')
```

Comment:
the difference has to do with variables, concatenation is adding strings together, interpolation is creating a string to include a variable with a string value (whether using `+` or `${}`)